react-native-aztec/src/AztecView.js

```diff
@@ -165,14 +181,15 @@ class AztecView extends React.Component {
           onContentSizeChange = { this._onContentSizeChange }
           onHTMLContentWithCursor = { this._onHTMLContentWithCursor }
           onSelectionChange = { this._onSelectionChange }
-          onEnter = { this.props.onEnter && this._onEnter }
+          onEnter = { this.props.onKeyDown && this._onEnter }
+          onBackspace = { this.props.onKeyDown && this._onBackspace }
```

Comment:
After the Android Aztec lib code is updated the `onEnter` and `onBackspace` props should be remove, and all "keypress" events should go troigh `onKeyDown`